tests/framework/db/oci/QueryBuilderTest.php

```diff
@@ -239,6 +239,10 @@ public function upsertProvider()
         foreach ($concreteData as $testName => $data) {
             $newData[$testName] = array_replace($newData[$testName], $data);
         }
+
+        // skip test
+        unset($newData['no columns to update']);
```

Comment:
Why skipping it? It was changed in OCI builder.

Reply:
Yes, but I don't have oci to run tests with it.

Reply:
Travis has it.

Reply:
I think it doesn't https://github.com/yiisoft/yii2/blob/master/.travis.yml#L31

Reply:
Oh, bad... Anyway, that's not the reason to exclude it.